var/spack/repos/builtin/packages/libint/package.py

```diff
@@ -195,3 +195,10 @@ def install(self, spec, prefix):
             configure(*options)
             make()
             make('install')
+
+    def patch(self):
+        # Use Fortran compiler to link the Fortran example, not the C++
+        # compiler
+        if '+fortran' in self.spec and self.spec.satisfies('%nvhpc'):
+            filter_file('\$\(CXX\) \$\(CXXFLAGS\)', '$(FC) $(FCFLAGS)',
+                        'export/fortran/Makefile')
```

Comment:
If you add `string=True` to these `filter_file` commands, it won't interpret the string as a regex so you won't have to escape things like parentheses.

Reply:
Thanks for the tip!  I've added it throughout.